src/clinical/donor-repo.ts

```diff
@@ -117,6 +121,25 @@ export const donorDao: DonorRepository = {
     return F(mapped);
   },
 
+  // Like findByProgramId, but DocQuery asks mongo to return PoJo without docIds for faster fetch
+  async findByProgramIdOmitMongoDocId(
```

Comment:
I found an optimization here. The download all takes about ~300ms for 3000 donors with sample registration and donor clinicalInfo.  I went a ahead and made this change, but we will probably need a more comprehensive look later.

Reply:
cool good to know this :+1: